src/sections/publication/view/publication-details-view.tsx

```diff
@@ -0,0 +1,324 @@
+
+---
+
+**¿Te ha resultado útil este video?** Compártelo con tus amigos o colegas que puedan estar interesados. Nos vemos en el próximo video, donde seguiremos explorando temas fascinantes y prácticos para tu crecimiento personal y profesional.
+
+¡Gracias por tu tiempo y hasta pronto! 🙌
+`;
+
+const MAX_LINES = 5;
+const SubscriptionPolicyContractAddress = '0xcafde3bc71b7ab469c8b165e896547d5868e9e5c'
+const GeoAddress = '0xEFBBD14082cF2FbCf5Badc7ee619F0f4e36D0A5B'
+
+
+// ----------------------------------------------------------------------
+
+type Props = {
+  id: string;
+};
+
+// ----------------------------------------------------------------------
+
+export default function PublicationDetailsView({ id }: Props) {
+  // STATES HOOKS
+  const [showToggle, setShowToggle] = useState(false);
+  const [showButton, setShowButton] = useState(false);
+  const descriptionRef = useRef(null);
+  // LOCAL HOOKS
+  const theme = useTheme();
+  const router = useRouter();
+  const { selectedProfile } = useAuth();
+  const mdUp = useResponsive('up', 'md');
+  // LENS HOOKS
+  const { data, loading }: any = usePublication({
+    forId: id as any,
+  });
+  // CONSTANTS
+  const variants = theme.direction === 'rtl' ? varFade().inLeft : varFade().inRight;
+
+  // CONTRACT
+  const isAccessAllowed: any = useReadContract({
+    abi: SubscriptionPolicyAbi.abi,
+    address: SubscriptionPolicyContractAddress,
+    functionName: 'isAccessAllowed',
+    args: [data?.by?.ownedBy?.address, selectedProfile?.ownedBy?.address], // first address is the address owner and the second one is the address who is asking if has access
+  })
+
+  const handleBack = () => {
+    router.push(paths.dashboard.root);
+  }
+
+  const getMediaUri = (cid: string): string => `https://ipfs.io/ipfs/${cid.replace('ipfs://', '')}`
+
+  const getWallpaperCid = (): string => data?.metadata?.attachments?.find((el: any) => el.altTag === 'Wallpaper')?.image?.raw?.uri
+
+  const getPosterCid = (): string => data?.metadata?.attachments?.find((el: any) => el.altTag === 'Vertical Poster')?.image?.raw?.uri
+
+  const toggleDescription = () => {
+    setShowToggle(!showToggle);
+  };
+
+  const handleSubscribe = () => {
+    // Subscribe logic
+  };
+
+  useEffect(() => {
+    if (descriptionRef.current) {
+      const lineHeight = parseInt(
+        window.getComputedStyle(descriptionRef.current).lineHeight,
+        10
+      );
+      const maxHeight = lineHeight * MAX_LINES;
+      if (descriptionRef.current.scrollHeight > maxHeight) {
+        setShowButton(true);
+      }
+    }
+  }, [descriptionRef.current, data?.metadata?.content]);
+
+  if (loading) return <LoadingScreen />
+
+  return (
+    <>
+      <Header>
```

Comment:
@Jadapema this header cause the issue when try to open the drawer in small screen. 
Its appeart to be on top of base header and the fire event not have to with the main behavior